src/styles/reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
```

Comment:
Further styling should be separated from reset into another file. Establishing a certain font shouldn't be mixed with reseting styles.

Reply:
I would also consider setting some rules on the body element as to easier position the `header` and `main` elements (especially that in the requirements we would like to have same padding on top and bottom of the main element). Try playing with `height` and `display: flex` properties